frontend/lit/Reference.js

```diff
@@ -8,7 +8,7 @@ class Reference {
         this.data = data;
         this._quickSearchText = `${data.title}-${data.year}-${data.authors}-${data.authors_short}`.toLowerCase();
         this.tags = data.tags.map(tagId => tagtree.dict[tagId]);
-        this.userTags = data.user_tags ? data.user_tags.map(tagId => tagtree.dict[tagId]) : null;
+        this.userTags = data.user_tags ? data.user_tags.map(tagId => tagtree.dict[tagId]) : [];
```

Comment:
@casey1173 do you remember why you made this change? 

This makes it so that if resolved tags exist, it suggests to the user to remove them?
![image](https://user-images.githubusercontent.com/999952/221894542-566d2d4f-8d80-466c-9061-3244fa241220.png)

I bet you added to fix something else, just curious if you remember what it was.

Reply:
> @casey1173 do you remember why you made this change?
> 
> This makes it so that if resolved tags exist, it suggests to the user to remove them? ![image](https://user-images.githubusercontent.com/999952/221894542-566d2d4f-8d80-466c-9061-3244fa241220.png)
> 
> I bet you added to fix something else, just curious if you remember what it was.

wow, interesting bug, nice catch! I made that change because I also started calling reference.userTags.length in TagReferences/Main.js to determine whether to add the new user tags icon. Accessing the length property of a null object caused an error

Reply:
ok, thanks, that makes sense. I think I've already fix that, so we should be good. The dual icons looks 💯 !